source/tests/consistent/model/test_zbl_ener.py

```
@@ -0,0 +1,224 @@
+            "fitting_net": {
+                "neuron": [
+                    5,
+                    5,
+                ],
+                "resnet_dt": True,
+                "precision": "float64",
+                "seed": 1,
+            },
+        }
+
+    tf_class = EnergyModelTF
```

Ruff

Undefined name `EnergyModelTF`

(F821)